ui/lib/core/addon/templates/components/replication-header.hbs

```diff
@@ -37,7 +37,7 @@
         {{if isSecondary 'Secondary' 'Primary'}}
       </span>
       <span class="tag" data-test-clusterId>
-        {{data.dr.clusterIdDisplay}}
+        {{secondaryId}}
```

Comment:
this doesn't have to be resolved in this PR, but i noticed some css inconsistencies while testing this part out in the UI. the mode & sha/label-thingy has slightly different colors in the 2 dashboards. additionally, the card headers have different font-weights:

![image](https://user-images.githubusercontent.com/903288/81753992-e7397200-9469-11ea-878e-4b0682e6886e.png)

it'd be good to pick one & go with it. i prefer the lighter grey label and the larger header title but let's summon @ivana-mcconnell and see what she thinks!

Reply:
update: sounds like the card headers should match the bolder style on the left, while the labels should have a background color of `#F7F8FA`. if it's a quick fix would you mind tackling that in this PR?

Reply:
Good catch.  I added a comment to the slack thread regarding this. I was able to change the background-color, but the grey mentioned by design for the text does not exist in the codebase.